src/utils/Constants.tsx

```diff
@@ -78,3 +79,346 @@ export const NAV_ITEMS = [
     nestedItems: [],
   },
 ]
+
+export const TABLE_DATA_HEADERS: column[] = [
+  { id: 1, value: '', type: 'checkbox', alignText: 'center' },
+  {
+    id: 2,
+    value: 'Journey Name',
+    type: 'sort',
```

Comment:
don't have type as sort.
add a property (sortable - true/false)
type should be "text/number/checkbox/ or others"